src/DemoData.hs

```diff
@@ -156,6 +187,9 @@ universe rnd = void $ do
 
 -- * Helpers
 
+-- This 'Monad m => m a' is strange, why not simply 'a' instead?
+-- Second, is this actually safe to use the same generator over and over or
+-- does it need to be threaded?
```

Comment:
Ok this is good enough for demo data but instead of split we should use a monad. I guess that Gen was getting in the way of persist. If one of them was a monad transformer we could easily combine the two.